01-node-tutorial/answers/content/12-http.js

```diff
@@ -0,0 +1,38 @@
+//built in module for building web server
+const http = require("http")
+
+//this server method creates a server, looks for callback
+//and takes in two parameters req and res the incoming
+//request and response is what method sends back
+//also need to define what port to listen to
+
+const server = http.createServer((req, res) => {
+  //lets console.log the req object
+  //notice how we dont see console.log because server is waiting for those req go to browser and refresh
+  //we see now a giant object we are looking of url
+  //property to see the endpoint client is requesting
+  console.log(req)
+  if (req.url === "/") {
+    res.end("Welcome to the party")
+  }
+  //sends different text
+  if (req.url === "/about") {
+    res.end("Here is our short history")
+  }
+  //what happens if user access resource that dont exist
+  //once we send response we have sent response we end request
+  res.end(`
```

Comment:
This needs to use `if/else if/else` when checking the url because it's not valid to call `res.end` more than once.

Reply:
Ah, I see will update thank Austin!